### ISSUE TITLE
Batch ops not supported?

### ISSUE THREAD

**Issue by apmcodes** [dekelev/feathers-cassandra#2]
Can anyone please confirm this do not support batch ops of express-cassandra models.doBatch.

While creating with array of 1000 items, getting the below error, seems insert ops are not batched but executed in parallel via promise.all .
```
name: 'BusyConnectionError',
        info:
         'Represents a client-side error indicating that all connections to a certain host have reached the maximum amount of in-flight requests supported (pooling.maxRequestsPerConnection)',
        message:
         'All connections to host 127.0.0.1:9042 are busy, 2048 requests are in-flight on a single connection'
```

**Comment by dekelev** [comment]
@apmcodes Good input. I've published v3.1.3 with batch create queries support. Check the new **$batch** query operator [here](https://www.npmjs.com/package/feathers-cassandra#special-query-operators).

**Comment by apmcodes** [comment]
That was a real quick update. Thanks.
